packages/cactus-plugin-ccmodel-hephaestus/src/test/typescript/fabric-contracts/lock-asset/chaincode-typescript/package.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "asset-transfer-basic",
```

Comment:
I think this is copied from https://github.com/hyperledger/fabric-samples/, could you please add a comment in each copied file with a link to the original sources (and possibly a date when we copied it)?